src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -189,6 +191,30 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void UpdateInstallerFileMotwIfApplicable(Execution::Context& context)
+    {
+        const std::string TrustedSourceIds[] = { "Microsoft.Winget.Source_8wekyb3d8bbwe" };

```

Comment:
>const std::string TrustedSourceIds[] = { "Microsoft.Winget.Source_8wekyb3d8bbwe" }; [](start = 8, length = 83)

SourceDetails should be added to so that we can set trust levels on arbitrary sources.